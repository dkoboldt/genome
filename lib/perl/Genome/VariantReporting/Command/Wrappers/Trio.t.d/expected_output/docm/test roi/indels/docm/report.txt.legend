Headers
chromosome_name	Chromosome: an identifier from the reference genome or an angle-bracketed ID String ("<ID>") pointing to a contig in the assembly file
start	The start position of the variant. One-based
stop	The end position of the variant. One-based, inclusive
reference	The reference allele at this position
variant	The variant called at this position
TEST-patient1-somval_normal1_per_library_var_count	Number of reads supporting variant for each library of sample TEST-patient1-somval_normal1
TEST-patient1-somval_tumor1_ref_count	Number of reads supporting the reference for sample TEST-patient1-somval_tumor1
TEST-patient1-somval_tumor1_per_library_ref_count	Number of reads supporting the reference for each library of sample TEST-patient1-somval_tumor1
TEST-patient1-somval_tumor2_ref_count	Number of reads supporting the reference for sample TEST-patient1-somval_tumor2
TEST-patient1-somval_tumor1_var_count	Number of reads supporting variant for sample TEST-patient1-somval_tumor1
TEST-patient1-somval_tumor2_vaf	Variant allele frequency for sample TEST-patient1-somval_tumor2
TEST-patient1-somval_tumor1_vaf	Variant allele frequency for sample TEST-patient1-somval_tumor1
TEST-patient1-somval_tumor2_per_library_vaf	Variant allele frequency for each library of sample TEST-patient1-somval_tumor2
TEST-patient1-somval_normal1_var_count	Number of reads supporting variant for sample TEST-patient1-somval_normal1
TEST-patient1-somval_normal1_ref_count	Number of reads supporting the reference for sample TEST-patient1-somval_normal1
TEST-patient1-somval_tumor1_per_library_var_count	Number of reads supporting variant for each library of sample TEST-patient1-somval_tumor1
TEST-patient1-somval_tumor2_per_library_var_count	Number of reads supporting variant for each library of sample TEST-patient1-somval_tumor2
TEST-patient1-somval_tumor2_var_count	Number of reads supporting variant for sample TEST-patient1-somval_tumor2
TEST-patient1-somval_normal1_vaf	Variant allele frequency for sample TEST-patient1-somval_normal1
TEST-patient1-somval_tumor2_per_library_ref_count	Number of reads supporting the reference for each library of sample TEST-patient1-somval_tumor2
TEST-patient1-somval_normal1_per_library_vaf	Variant allele frequency for each library of sample TEST-patient1-somval_normal1
TEST-patient1-somval_normal1_per_library_ref_count	Number of reads supporting the reference for each library of sample TEST-patient1-somval_normal1
TEST-patient1-somval_tumor1_per_library_vaf	Variant allele frequency for each library of sample TEST-patient1-somval_tumor1
Filters
