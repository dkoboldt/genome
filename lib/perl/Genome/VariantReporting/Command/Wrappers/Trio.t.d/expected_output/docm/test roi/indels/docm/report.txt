chromosome_name	start	stop	reference	variant	TEST-patient1-somval_normal1_per_library_var_count	TEST-patient1-somval_tumor1_ref_count	TEST-patient1-somval_tumor1_per_library_ref_count	TEST-patient1-somval_tumor2_ref_count	TEST-patient1-somval_tumor1_var_count	TEST-patient1-somval_tumor2_vaf	TEST-patient1-somval_tumor1_vaf	TEST-patient1-somval_tumor2_per_library_vaf	TEST-patient1-somval_normal1_var_count	TEST-patient1-somval_normal1_ref_count	TEST-patient1-somval_tumor1_per_library_var_count	TEST-patient1-somval_tumor2_per_library_var_count	TEST-patient1-somval_tumor2_var_count	TEST-patient1-somval_normal1_vaf	TEST-patient1-somval_tumor2_per_library_ref_count	TEST-patient1-somval_normal1_per_library_vaf	TEST-patient1-somval_normal1_per_library_ref_count	TEST-patient1-somval_tumor1_per_library_vaf
1	2013	2013	A	T	.	.	.	.	.	.	.	.	.	.	.	.	.	.	.	.	.	.
